components/linkPrefetch/index.js

```diff
@@ -0,0 +1,135 @@
+import { Toggle, TextField, SelectField, Container } from "@newfold/ui-component-library";
+
+const LinkPrefetch = ({methods, constants}) => {
+	const [settings, setSettings] = methods.useState(methods.NewfoldRuntime.sdk.linkPrefetch.settings);
+	const [isError, setIsError] = methods.useState(false);
+	const apiUrl = methods.NewfoldRuntime.createApiUrl("/newfold-ecommerce/v1/linkprefetch/update");
+
+	const handleChangeOption = ( option, value ) => {
+		if ( option in settings ) {
+			const updatedSettings = settings;
+			updatedSettings[option] = value;
+			methods.apiFetch({
+				url: apiUrl,
+				method: "POST",
+				data: {settings: updatedSettings}
+			  }).then((result)=>{
+				  setSettings( (prev)=> {
+					return {
+						...prev,
+						[option]: value
+					}
+				});
+			  }).catch((error) => {     
+				setIsError(error.message);  
+			  });
+		}
+	}
+
+	methods.useUpdateEffect(() => {
+        methods.setStore({
+            ...constants.store,
+            linkPrefetch: settings,
+        });
+
+        methods.makeNotice(
+            "link-prefetch-change-notice", 
+            constants.text.linkPrefetchTitle,
+            !isError ? constants.text.linkPrefetchNoticeTitle : isError,
+            !isError ? "success" : "error",
+            5000
+        );
+    }, [settings, isError]);
+
+	return(
+		<>
+		<Container.SettingsField
+			title={constants.text.linkPrefetchTitle}
+			description={constants.text.linkPrefetchDescription}
+		>
+			<div className="nfd-toggle-field nfd-mb-6" style={{display: 'flex', flexDirection:'row'}}>
```

Comment:
Let's move any CSS `styles` to a new `styles.css` and enqueue that via PHP.

Reply:
Ok so I will a create a new styles.css and move it in a styles folder maybe and load it from Performance.php on admin side only. I think it could be better to load it in a main class so it could be used to add styles to other settings if needed. I will commit this then you will let me know if ok or not ;)

Reply:
at the end I added two new functions in functions.php getStylesPath and getScriptsPath so they could be used generally when needed and are not related to a single class